docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -380,15 +380,26 @@ V&V Checks:
     - XX relative risk on incidence of encephalopathy if birthing parent experiences obstructed labor
     - Need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality
   * - 7
-    - XX% of each type of facility have CPAP or NICU capabilities
-    - 
+    - Percentage of each facility type that have CPAP capabilities: CMONC - 39.3% and BMONC - 7.5% 
+    - These values are from the 2016 EmONC Final Report and are therefore only reflective of Ethiopian health systems. Please use these as a placeholder for now while we 
+      try to find reliable values for Nigeria and Pakistan. 
   * - 8
-    - XX relative risk on RDS mortality of facility having CPAP or NICU capabilities vs. not
-    - Need to confirm this will impact mortality not incidence. Also need to determine how neonatal mortality in general will be modeled and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality
+    - 0.64 (95% CI 0.50-0.82) relative risk on RDS mortality of facility having CPAP capabilities 
+    - Need to confirm this will impact mortality not incidence. Also need to determine how neonatal mortality in general will be modeled 
+      and how we will handle overlaps with preterm and LBWSG RR's on all cause mortality. Note that we might want RR for NICU, but this value 
+      is for CPAP capabilities. Source: `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_
```

Comment:
```suggestion
    - Note that we might want RR for NICU, but this value 
      is for CPAP capabilities. Source: `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_
```
The number you cite does in fact impact mortality, not incidence, right? And we've figured out how to model neonatal mortality, preterm, etc., so that sentence can be deleted.